internal/pkg/template/templates/workloads/partials/cf/nlb.yml

```diff
@@ -11,57 +11,58 @@ PublicNetworkLoadBalancer:
             !Sub '${AppName}-${EnvName}-PublicSubnets'
     Type: network
 
-NLBListener:
+{{- range $listener := .NLB.Listener}}
+NLBListenerFor{{ $listener.Port }}:
```

Comment:
Why do we need so many listeners...can we not have one listener with multiple target groups?

Reply:
In NLB, we have a one-to-one mapping of nlb port vs. container port. Due to that reason, we have to create an NLBListener for each routing rule. Also, as opposed to the public load balancer, which has multiple listener rules per ALB listener, NLB doesn't have that facility. 